wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/DifferentialDrivetrainSim.java

```diff
@@ -416,8 +416,16 @@ public void setPose(Pose2d pose) {
 
   /** Represents common wheel sizes of the kit drivetrain. */
   public enum KitbotWheelSize {
+    kFourInch(Units.inchesToMeters(4)),
```

Comment:
Is there actually a four inch KOP wheel size?

Reply:
Not according to [the 2021 EveryoneBox](https://firstfrc.blob.core.windows.net/frc2021/KitofParts/2021KOPChecklist-EveryoneBox.pdf)
I believe some teams are using 4" wheels outside of the KOP tho.

Reply:
If we add 4” here we need to add it to C++ as well.  I’d prefer to make that a separate PR since it’s a separate functional change, so can you pull it out of this PR?

Reply:
Done.
Am I working against the correct branch? Should I be doing this against 2022?

Reply:
Working against main is fine.  We are planning on merging the 2022 branch into main relatively soon.